lib/core/token_requests.rb

```diff
@@ -5,17 +5,17 @@ class AccessTokenRequest
     def initialize(environment, refreshToken = nil)
```

Comment:
Something else that could help be more ruby-esque would be using snake_case in favor of camelCase.

```suggestion
    def initialize(environment, refresh_token = nil)
```